pkg/pool-stable-phantom/contracts/StablePhantomPool.sol

```diff
@@ -877,23 +899,24 @@ contract StablePhantomPool is IRateProvider, BaseGeneralPool, StablePoolStorage,
         _updateOldRates();
     }
 
+    function _getPostJoinExitAmp() internal view returns (uint256) {
+        return _postJoinExitAmp;
+    }
+
+    function _getPostJoinExitInvariant() internal view returns (uint256) {
+        return _postJoinExitInvariant;
+    }
```

Comment:
These are not immutable variables, so we don't really gain much from a getter. Specially considering we use them in a single place.